package.json

```diff
@@ -21,8 +21,6 @@
   "license": "MIT",
   "files": [
     "canvg.js",
-    "rgbcolor.js",
```

Comment:
you don't need these files, they're included by NPM

Reply:
agree on this